src/Microsoft.AspNetCore.HttpOverrides/ForwardedHeadersMiddleware.cs

```diff
@@ -17,9 +17,18 @@ namespace Microsoft.AspNetCore.HttpOverrides
 {
     public class ForwardedHeadersMiddleware
     {
-        private const string XForwardedForHeaderName = "X-Forwarded-For";
-        private const string XForwardedHostHeaderName = "X-Forwarded-Host";
-        private const string XForwardedProtoHeaderName = "X-Forwarded-Proto";
+        private string XForwardedForHeaderName
+        {
+            get { return _options.ForwardedForHeaderName; }
+        }
+        private string XForwardedHostHeaderName
+        {
+            get { return _options.ForwardedHostHeaderName; }
+        }
+        private string XForwardedProtoHeaderName
```

Comment:
style: either remove these and access options directly, or at least move them down after the constructor